src/styles/abstracts/_variables.scss

```
@@ -0,0 +1,3 @@
+.responsive-padding {
+    padding: 0 15px;
+}
```

```suggestion
$responsive-px: 15px;
```

Thank you for creating `_variables.scss`🙂🙏 But I meant to create a variable something like this, so could you change it and import the variable to `_profile.scss` instead of having the class `.responsive-padding`?

---

Sorry for my misunderstanding, and not sure If I fixed correctly, but could you check it again?